hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/generator/SqlStreamGeneratorTest.java

```diff
@@ -38,35 +38,40 @@ public static void setUpClass() {
     @Test
     public void test_generateStream() {
         assertRowsEventuallyInAnyOrder(
```

Comment:
This is likely to fail intermittently if there's some hiccup. We could add a method that will ignore unexpected rows.